index.html

```diff
@@ -0,0 +1,69 @@
+<!-- <img class="Ellipse12" src="images/Ellipse 12.png" alt="">
+            <img class="Ellipse13" src="images/Ellipse 13.png" alt="">
+            <img class="Ellipse4" src="images/Ellipse 4.png" alt="">
+            <img class="Rectangle5" src="images/Rectangle 5.png" alt="">
+            <img class="Ellipse3" src="images/Ellipse 3.png" alt="">
+            <img class="Rectangle4" src="images/Rectangle 4.png" alt="">
+            <img class="Rectangle7" src="images/Rectangle 7.png" alt="">
+            <img class="Ellipse5" src="images/Ellipse 5.png" alt="">
+            <img class="Ellipse11" src="images/Ellipse 11.png" alt="">
+            <img class="Rectangle6" src="images/Rectangle 6.png" alt="">
+            <img class="Ellipse15pro" src="images/Ellipse15pro.png" alt="">-->
+
```

Comment:
As mentioned by the previous reviewer, you are supposed to remove this code from your .html file and not comment them out. Aside making your project unproffessional, it can also make it bloated and slow. 😉